PowerPointLabs/PowerPointLabs/ShapesLab/Views/ShapesLabCategoryInfoDialogBox.xaml.cs

```diff
@@ -28,6 +29,9 @@ public partial class ShapesLabCategoryInfoDialogBox
         // Regex = [<>:"/\\|?*]
         private const string InvalidCharsRegex = "[<>:\"/\\\\|?*]";
```

Comment:
it might be better to include the `ReservedNameList` in this regex expression so that only one check is needed below!